aws-integration/trace-sample-app/values.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  ports:
+    - name: "4567"
+      port: 4567
+      targetPort: 4567
+  selector:
+    io.kompose.service: sample-app
+status:
+  loadBalancer: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: sample-app
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: sample-app
+    spec:
+      containers:
+        - env:
+            - name: AWS_REGION
+              value: ${AWS_REGION}
+            - name: LISTEN_ADDRESS
+              value: 0.0.0.0:4567
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: http://otel-collector.${KYMA_NS}:4317
+            - name: OTEL_RESOURCE_ATTRIBUTES
+              value: service.namespace=${KYMA_NS},service.name=otel-collector
```

Comment:
The service name `otel-collector` does not make so much sense. Ususally you want to use this to identify your application. Environment variables are again not effective here.